### PR TITLE
Clarification for elements

### PR DESCRIPTION
**Pull request by SteveLLamb**

### CODE REVIEW

**Review by palemieux — CHANGES_REQUESTED** [review]

File: doc/main.html [hunk 1]
```diff
@@ -608,12 +608,18 @@ <h4>Additional elements section</h4>
 
         <p>Each <code>a</code> element shall have:</p>
         <ul>
-          <li>an <code>id</code> attribute whose value is prefixed with <code>element-</code>;</li>
+          <li>an <code>id</code> attribute whose value is prefixed with <code>element-</code>, and value itterated, starting with <code>a</code> ;</li>
```
[the remark this answers is not in the document]
Comment (SteveLLamb) [reply doc/main.html:611]: This was how we came up with the naming convention for element repos, based on the `<ol type="a"><li>` item. This makes it easier to keep that consistent IMO.

[the remark this answers is not in the document]
Comment (palemieux) [reply doc/main.html:611]: I do not think we should/can require that the editor potentially modify *all* the `id` values whenever an additional element is removed.

[the remark this answers is not in the document]
Comment (palemieux) [reply doc/main.html:611]: See https://github.com/SMPTE/github-operating-manual/pull/9#discussion_r1257547247

**Review by palemieux — CHANGES_REQUESTED** [review]

File: doc/main.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (SteveLLamb) [reply doc/main.html:611]: agreed as per https://github.com/SMPTE/github-operating-manual/pull/9#discussion_r1258571497, I will updates as noted if you agree.

[the remark this answers is not in the document]
Comment (SteveLLamb) [reply doc/main.html]: Addressed in https://github.com/SMPTE/html-pub/pull/137/commits/f83fba97f010acad995d25518031cb13f4835396

File: doc/main.html [hunk 2]
```diff
@@ -610,15 +610,21 @@ <h4>Additional elements section</h4>
         <ul>
           <li>an <code>id</code> attribute whose value is prefixed with <code>element-</code>;</li>
           <li>a <code>title</code> attribute that provides a short description of the element;</li>
-          <li>a <code>href</code> attribute that links to the element itself, typically in the <code>doc/elements</code>
-          directory.</li>
+          <li>a <code>href</code> attribute that links to the element itself</li>
```
Comment (palemieux) [inline doc/main.html:613]:
```suggestion
          <li>a <code>href</code> attribute that links to the element, as detailed below.</li>
```

File: doc/main.html [hunk 3]
```diff
@@ -610,15 +610,21 @@ <h4>Additional elements section</h4>
...
         </ul>
 
+        <p>
```
Comment (palemieux) [inline doc/main.html:616]: Each additional element shall be managed in one of the following ways:

* as a file in the doc/elements directory, in which case the link to the element shall be relative path to the file, e.g. "elements/element.txt".
* in a separate SMPTE repository, in which case the link to the element shall be a versioned link to the repo, e.g. https://github.com/SMPTE/html-pub/tree/631ed7a0312fa183c356a7b742129ce46f92736f

**Review by palemieux — CHANGES_REQUESTED** [review]

File: doc/main.html (hunk 3 above)
Comment (SteveLLamb) [reply doc/main.html:616]: updated in https://github.com/SMPTE/html-pub/pull/137/commits/b72678433517467034a0ee22a61d202f3d65e3a3

File: doc/main.html (hunk 2 above)
Comment (SteveLLamb) [reply doc/main.html]: updated in https://github.com/SMPTE/html-pub/pull/137/commits/b72678433517467034a0ee22a61d202f3d65e3a3

File: doc/main.html [hunk 4]
```diff
@@ -610,15 +610,26 @@ <h4>Additional elements section</h4>
...
+          Each additional element shall be managed in one of the following ways:
+        </p>
+        <ul>
+          <li>as a file in the doc/elements directory, in which case the link to the element shall be relative path to the file, e.g. <code>.elements/element.txt</code></li>
+          <li>in a separate SMPTE repository, in which case the link to the element shall be a versioned link to the repo, e.g. <code>https://github.com/SMPTE/html-pub/tree/631ed7a0312fa183c356a7b742129ce46f92736f</code></li>
+        </ul>
+
+        <p class="note">
+          It is advised to only referenced element repos managed by SMPTE to fullfil license requirements, see <a href="#bib-smpte-gh-om"></a> for more details. If an element has been created outside of SMPTE, it should be considered as a bibliographic reference instead.
```
Comment (palemieux) [inline doc/main.html:625]:
```suggestion
          Information that is not an integral part of the document is cited using a normative or bibliographic reference. 
```

**Review by palemieux — CHANGES_REQUESTED** [review]

File: doc/main.html (hunk 4 above)
Comment (SteveLLamb) [reply doc/main.html]: adressed in https://github.com/SMPTE/html-pub/pull/137/commits/f8b7af5b54ab6fa653cca071cecbd16e482084c1

Comment (SteveLLamb) [reply doc/main.html]: Better in https://github.com/SMPTE/html-pub/pull/137/commits/73f2f5cfd27399c64489aca14a61682c38361991

**Review by palemieux — APPROVED** [review]

**Pull request merged**